docs/aboutstudio/features.mdx

```diff
@@ -10,9 +10,9 @@ The following provides a high-level overview of the Use Cases and Methods you ca
 
 | Feature | Description  | Capabilities   |
 | ------------ | ------------ | ------------ |
-|  Livestream  |   | Multistream, Livestream Recording   |
-|  Upload  | Video on Demand (VOD)    | Export to IPFS, Upload video to Livepeer Studio (API or Dashboard)  |
-|  Mint | Minting mp4 video as an NFT   |  Mint mp4 video files, add your smart contract to mint video nft  |
+|  Livestream  | Livestreaming using broadcasting software or WebRTMP  | Multistream, Livestream Recording, Suspend streams  |
+|  Upload  | Upload and store videos for On Demand access | Export to IPFS or other services, Upload video to Livepeer Studio (API or Dashboard), Store assets on decentralized storage  |
+|  Mint | Minting `.mp4` video as an NFT   |  Mint `.mp4` video files, Use to create custom ERC-721 smart contracts, Deploy to mainnet, testnet or layer 2 solutions like Polygon in you custom smart contract |
```

Comment:
wondering if we can ake the list into a list of bullets, e.g., 
- Multisream
- Livestream Recording,
- Suspend streams
same for all the rows and features

Reply:
@hnjolles1 converted into lists 👇